master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -29,10 +29,18 @@ We'll use a new namespace for this guide.  Run the following command to create i
 kubectl create ns advanced-policy-demo
 ```
 
-And then enable isolation on the Namespace.
+And then enable isolation on the Namespace using a [default policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/#default-policies).
 
 ```
-kubectl annotate ns advanced-policy-demo "net.beta.kubernetes.io/network-policy={\"ingress\":{\"isolation\":\"DefaultDeny\"}}"
+kubectl create -f - <<EOF
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
```

Comment:
Should these stay at extensions/v1beta1? They could be networking.k8s.io/v1 (I believe) but then that would only work for k8s 1.7. Seems reasonable to stay as they are, I just wanted to put the thought out there.  We could have different blocks for 1.6 and 1.7, that would make it obvious at a later point that the old apiVersion needed to be cleaned up.